src/api/operator/operatorApi.ts

```diff
@@ -204,17 +206,9 @@ export async function getOrders(params: GetOrdersParams): Promise<RawOrder[]> {
     `[getOrders] Fetching orders on network ${networkId} with filters: owner=${owner} sellToken=${sellToken} buyToken=${buyToken}`,
   )
 
-  const searchString = new URLSearchParams(
-    Object.keys(searchParams).reduce((acc, key) => {
-      // Pick keys that have values non-falsy
-      if (searchParams[key]) {
-        acc[key] = encodeURIComponent(searchParams[key])
-      }
-      return acc
-    }, {}),
-  ).toString()
-
-  const queryString = '/orders/?' + searchString
+  const searchString = buildSearchString({ ...searchParams })
```

Comment:
types issue? i feel like this is just avoiding the problem but :roll_eyes: if it works, it works eh

Reply:
:shrug: